lib/txgh/github_repo.rb

```diff
@@ -19,6 +19,22 @@ def process_all_branches?
       branch == 'all'
     end
 
+    def should_process_branch?(candidate)
+      process_all_branches? ||
+        candidate.include?(github_config_branch) ||
+        candidate.include?('L10N')
```

Comment:
Yeah... I'm actually not sure what this is for. We inherited it from Transifex. Maybe I should just get rid of it in our fork?